dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormField.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <template v-if="evidenceDesired">
```

Comment:
"evidenceDesired" seems to always be "true" for all DataPointFormField mounts.
So this v-if is not really needed currently.

Reply:
Removed

Reply:
👍